spring-hibernate5/src/main/java/com/baeldung/spring/PersistenceXmlConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.spring;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ImportResource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Configuration
+@EnableTransactionManagement
+@ComponentScan({ "com.baeldung.persistence.dao", "com.baeldung.persistence.service" })
+@ImportResource({ "classpath:hibernate5Config.xml" })
+public class PersistenceXmlConfig {
+
+    public PersistenceXmlConfig() {
```

Comment:
Same as before - don't need to add a no-arg constructor and invoke super() if the class does not extend another class (other than Object).